goshopify.go

```diff
@@ -424,3 +431,14 @@ func (c *Client) Put(path string, data, resource interface{}) error {
 func (c *Client) Delete(path string) error {
 	return c.CreateAndDo("DELETE", path, nil, nil, nil)
 }
+
+func GetAdminVersionedApiPathPrefix() string {
+	// 	if our apiVersion exists, this should return the form:  "admin/api/2019-04"
+	// 	if not, this should return the form:  "admin/", as the default year ago version going forward.
+	var rxPat = regexp.MustCompile(`^[0-9]{4}-[0-9]{2}$`)
+	if len(apiVersion) > 0 && rxPat.MatchString(apiVersion) {
+		return fmt.Sprintf("admin/api/%s", apiVersion)
+	} else {
+		return "admin"
```

Comment:
This line will also need to get hit with a test (that's why the coverage dropped).

Reply:
Kind of skirted this by getting rid of the function, probably should expand the NewClient test to include invalid version input testing.